File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 3 - Python/q15.py
#Segundo uma tabela médica, o peso ideal está relacionado com a altura e sexo. Faça um algoritmo que
#receba a altura e o sexo (M ou F) de uma pessoa, calcule e imprima o seu peso ideal, utilizando as seguintes fórmulas:
#Homens: (72.7 * altura) – 58
#Mulheres: (62.1 * altura) – 44.7

altura=float(input("Insira a altura da pessoa: "))
sexo=input("Insira o M(Para Masculino) e F(Para Feminino)")
if sexo=="M" or sexo=="m":
    peso=(72.7*altura)-58
    print("Peso da pessoa: ", peso)
elif sexo=="F" or sexo=="f":
    peso=(62.1*altura)-44.7
    print("Peso da pessoa: ", peso)
else:
    print("Opção Inválida")
        

File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 3 - C/q15.c
//Segundo uma tabela médica, o peso ideal está relacionado com a altura e sexo. Faça um algoritmo que
//receba a altura e o sexo (M ou F) de uma pessoa, calcule e imprima o seu peso ideal, utilizando as seguintes fórmulas:
//Homens: (72.7 * altura) – 58
//Mulheres: (62.1 * altura) – 44.7

#include<stdio.h>
int main(){
	float altura, peso;
	int sexo;
	printf("Altura: ");
	scanf("%f", &altura);
	printf("\n1 - Masculino \n2 - Feminino \nSexo: ");
	scanf("%d", &sexo);
	if(sexo==1){
		peso=(72.7*altura)-58;
    	printf("Peso da pessoa: %f", peso);
	}
	else if(sexo==2){
		peso=(62.1*altura)-44.7;
    	printf("Peso da pessoa: %f", peso);
	}
}
